lib/kwtool/accounts/user.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Kwtool.Accounts.User do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Argon2
+
+  schema "users" do
+    field :email, :string
+    field :full_name, :string
+    field :company, :string
+    field :encrypted_password, :string
+    field :password, :string, virtual: true
+    field :password_confirmation, :string, virtual: true
+
+    timestamps()
+  end
+
+  def changeset(user \\ %__MODULE__{}, attrs) do
```

Comment:
I see you are using this function in `Kwtool.Accounts.change_user/2` so when we update user information, we need to pass all the attributes every time (including `password` and `password_confirmation`). It's better to pass only the attribute that we want to update.

How about we separate this changeset to `create_changeset/2` and `update_changeset/2`? 
```elixir
# For create changeset, we validate all the required attributes for the user.
def create_changeset(user \\ %__MODULE__{}, attrs) do
    user
    |> cast(attrs, [:email, :full_name, :company, :password, :password_confirmation])
    |> validate_required([:email, :full_name, :password, :password_confirmation])
    |> unique_constraint(:email)
    |> validate_format(:email, ~r/@/)
    |> validate_length(:password, min: 3, max: 100)
    |> validate_confirmation(:password)
    |> hash_password()
end

# For update changeset, sometime we want to update some of the attributes. 
# So we don't need to validate presence of attributes.
def update_changeset(user \\ %__MODULE__{}, attrs) do
    user
    |> cast(attrs, [:email, :full_name, :company, :password, :password_confirmation])
    |> unique_constraint(:email)
    |> validate_format(:email, ~r/@/)
    |> validate_length(:password, min: 3, max: 100)
    |> validate_confirmation(:password)
    |> hash_password()
end
```